front/lib/plans/usage/mau.ts

```diff
@@ -8,7 +8,7 @@ import type { MauReportUsageType } from "@app/lib/plans/usage/types";
 import { InvalidRecurringPriceError } from "@app/lib/plans/usage/types";
 import { getFrontReplicaDbConnection } from "@app/lib/resources/storage";
 
-export async function getActiveUsersInWorkspaceForPeriod({
+async function countActiveUsersForPeriodInWorkspace({
```

Comment:
nit: the function signature makes the InWorkspace clear so no need to have it in the title IMHO

Reply:
I'll rename when moving those to the `WorkspaceResource`.